plugins/evernote.koplugin/main.lua

```diff
@@ -36,11 +37,21 @@ function EvernoteExporter:init()
     self.evernote_username = settings.username or ""
     self.evernote_token = settings.token
     self.notebook_guid = settings.notebook
+    self.joplin_IP = settings.joplin_IP or "localhost"
+    self.joplin_port = settings.joplin_port or 41185
+    self.joplin_token = settings.joplin_token -- or your token
+    self.joplin_notebook_guid = settings.joplin_notebook_guid or nil
     self.html_export = settings.html_export or false
+    self.joplin_export = settings.joplin_export or false
+    self.txt_export = settings.txt_export or false
+    ---@todo Is this if block necessarry? Nowhere in the code they are assigned both true.
+    --Do they check against external modifications to settings file?
```

Comment:
```suggestion
    -- Do they check against external modifications to settings file?
```

Our comments like a space. ;-)